feature-home/src/main/java/cm/aptoide/pt/feature_home/presentation/ViewModelProvider.kt

```diff
@@ -0,0 +1,43 @@
+package cm.aptoide.pt.feature_home.presentation
+
+import androidx.appcompat.app.AppCompatActivity
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.platform.LocalContext
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.ViewModelProvider.Factory
+import androidx.lifecycle.viewmodel.compose.viewModel
+import cm.aptoide.pt.aptoide_network.domain.UrlsCache
+import cm.aptoide.pt.feature_home.domain.BundlesUseCase
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
+import kotlin.reflect.KFunction0
+
+@HiltViewModel
+class InjectionsProvider @Inject constructor(
+  val urlsCache: UrlsCache,
+  val bundlesUseCase: BundlesUseCase,
+) : ViewModel()
+
+@Composable
+fun bundlesList(key: String?): Pair<BundlesViewUiState, KFunction0<Unit>> {
+  val injectionsProvider = hiltViewModel<InjectionsProvider>()
+  val vm: BundlesViewModel = viewModel(
+    viewModelStoreOwner = LocalContext.current as AppCompatActivity,
```

Comment:
You don't need this until you want to reuse the same view model instance on different screens (navigation routes)